src/geofr/utils.py

```diff
@@ -47,7 +47,8 @@ def extract_perimeters_from_file(perimeter_list_file):
             raise Exception(msg)
 
     # check for duplicates
-    duplicates = [item for item, count in collections.Counter(item_list).items() if count > 1]  # noqa
+    duplicates = [item for item, count in \
```

Comment:
J'avais mergé trop vite finalement, `make checkstyle` m'a renvoyé 2 erreurs sur cette modif
- ./geofr/utils.py:50:43: E502 the backslash is redundant between brackets
- ./geofr/utils.py:51:9: E128 continuation line under-indented for visual indent

comme quoi ça serait plus simple si c'était fait à notre place 😁 @lazybird